common/render_shaders.go

```diff
@@ -285,7 +287,18 @@ func (s *basicShader) generateBufferContent(ren *RenderComponent, space *SpaceCo
 
 	tint := colorToFloat32(ren.Color)
 
-	u, v, u2, v2 := ren.Drawable.View()
+	//u, v, u2, v2 := ren.Drawable.View()
+	u := float32(0)
+	v := float32(0)
```

Comment:
you can assign `u` and `v` to something like this: `var u,v float32` (https://play.golang.org/p/LnCUPb50j-4)